presto-spi/src/main/java/com/facebook/presto/spi/type/SqlDate.java

```diff
@@ -60,8 +57,6 @@ public boolean equals(Object obj)
     @Override
     public String toString()
     {
-        SimpleDateFormat format = new SimpleDateFormat("yyyy-MM-dd");
-        format.setTimeZone(getTimeZoneForKey(UTC_KEY));
-        return format.format(new Date(TimeUnit.DAYS.toMillis(days)));
+        return LocalDate.ofEpochDay(days).format(DateTimeFormatter.ofPattern("YYYY-MM-dd").withZone(ZoneId.of("UTC")));
```

Comment:
Calling `withZone` when formatting a temporal object that does not contain an instant is no-op unless you are printing out the offset. Quote from java doc:

> If the temporal object does not contain an instant, but does contain an offset then an additional check is made. If the normalized override zone is an offset that differs from the offset of the temporal, then a DateTimeException is thrown. In all other cases, the override zone is added to the temporal, replacing any previous zone, but without changing the date/time.